samples/apps/copilot-chat-app/webapi/DeploymentTemplates/DeploySK-Existing-OpenAI.ps1

```diff
@@ -0,0 +1,87 @@
+<#
+.SYNOPSIS
+Creates a Semantic Kernel service deployment using an existing OpenAI account.
+#>
+
+param(
+    [Parameter(Mandatory)]
+    [string]
+    # Name for the deployment
+    $DeploymentName,
+
+    [Parameter(Mandatory)]
+    [string]
+    # OpenAI API key
+    $ApiKey,
+
+    [string]
+    # Model to use for chat completions
+    $CompletionModel = "gpt-3.5-turbo",
+
+    [string]
+    # Model to use for text embeddings
+    $EmbeddingModel = "text-embedding-ada-002",
+
+    [string]
+    # Completion model the task planner should use
+    $PlannerModel = "gpt-3.5-turbo",
+
+    [Parameter(Mandatory)]
+    [string]
+    # Subscription to which to make the deployment
+    $Subscription,
+
+    [string]
+    # Resource group to which to make the deployment
+    $ResourceGroup = "",
+
+    [string]
+    # Region to which to make the deployment (ignored when deploying to an existing resource group)
+    $Region = "South Central US",
+
+    [string]
+    # Package to deploy to web service
+    $PackageUri = 'https://skaasdeploy.blob.core.windows.net/api/skaas.zip',
```

Comment:
Remind me again - are we maintaining skaasdeploy blob store with the skaas.zip file?